src/call/mod.rs

```diff
@@ -25,6 +28,36 @@ const BUF_SHRINK_SIZE: usize = 4 * 1024;
 #[derive(PartialEq, Clone, Copy, Debug)]
 pub struct RpcStatusCode(i32);
 
+impl Display for RpcStatusCode {
+    fn fmt(&self, fmt: &mut Formatter<'_>) -> fmt::Result {
+        write!(
+            fmt,
+            "{}-{}",
+            self.0,
+            match self.0 {
+                0 => "OK",
+                1 => "CANCELLED",
+                2 => "UNKNOWN",
+                3 => "INVALID_ARGUMENT",
+                4 => "DEADLINE_EXCEEDED",
+                5 => "NOT_FOUND",
+                6 => "ALREADY_EXISTS",
+                7 => "PERMISSION_DENIED",
+                8 => "RESOURCE_EXHAUSTED",
+                9 => "FAILED_PRECONDITION",
+                10 => "ABORTED",
+                11 => "OUT_OF_RANGE",
+                12 => "UNIMPLEMENTED",
+                13 => "INTERNAL",
+                14 => "UNAVAILABLE",
+                15 => "DATA_LOSS",
+                16 => "UNAUTHENTICATED",
+                _ => "INVALID_STATUS_CODE",
```

Comment:
How about adding the number to the message?

Reply:
The number is always added to the message (see line 35 and second arg of format), so the messages will be `13-INTERNAL` or `35-INVALID_STATUS_CODE`